src/main/java/com/openlattice/chronicle/ChronicleStudyApi.java

```diff
@@ -91,6 +93,30 @@ Boolean isKnownParticipant(
             @Path( STUDY_ID ) UUID studyId,
             @Path( PARTICIPANT_ID ) String participantId );
 
+    /**
+     * Delete a participant and their data.  Returns the number of entities removed.
+     *
+     * @param studyId       - studyId
+     * @param participantId - participant id
+     */
+    @DELETE( BASE + STUDY_ID_PATH + PARTICIPANT_ID_PATH )
+    Integer deleteParticipantAndAllNeighbors(
```

Comment:
A return value here implies this is a blocking call. I don't think it should be a blocking call.